### DISCUSSION TITLE
Help! The way to add interactions on Spoke

### DISCUSSION THREAD

**Discussion by SBleeyouk** [mozilla/hubs#4674] [Spoke Questions]
Hi, Are there any methods to add interactions on 3D objects? I'm trying to make interactions such as
- Embed pdf files into 3D book object, and when users click the next button, the pages go over like the real book.
- Make a bot that tells the mission or instructions how to use virtual space. There is a question mark button above the 3D character and when the user clicks this button, the text comes out.
I've found that customizing interactions is available by changing the console codes, but is it the only way to do it?
Also, I want to know how to add scripts on 3D objects.

**Comment by pattersonbl2** [comment]
This question would be great to also ask in our community discord. You can join here (https://discord.com/invite/dFJncWwHun). I will ask my team if they may have a quick answer as well.
